src/main/kotlin/com/faforever/tournamentlauncher/messaging/LobbyDTO.kt

```diff
@@ -54,26 +53,36 @@ fun DomainMatchParticipant.toLobbyDtoMatchParticipant() = MatchParticipant(
 )
 
 data class MatchCreateRequest(
-    @JsonIgnore
+    @JsonProperty("request_id")
     val requestId: UUID,
     @JsonProperty("game_name")
     val gameName: String,
     @JsonProperty("map_name")
     val mapName: String,
-    @JsonProperty("matchmaker_queue")
-    val matchmakerQueue: String,
+    @JsonProperty("featured_mod")
+    val featuredMod: String, // TODO:enum
+    @JsonProperty("game_options")
+    val gameOptions: Map<String, String>,
     val participants: List<MatchParticipant>,
 )
 
 data class MatchCreateSuccess(
     @JsonProperty("game_id")
     val gameId: Int,
 )
-
+enum class MatchCreateErrorCode {
+    PLAYER_NOT_ONLINE,
+    PLAYER_NOT_IDLE,
+    PLAYER_NOT_CONFIRMING,
+    PLAYER_NOT_STARTING,
+    PLAYER_NOT_CONNECTING,
+    OTHER
+}
```

Comment:
If these errors are create errors only, I'd put the enum inside the MatchCreateError class